source/mainnet/net/web3-id/concordia.rst

```diff
@@ -0,0 +1,32 @@
+.. _concordia:
+.. include:: ../../variables.rst
+
+=========
+Concordia
+=========
+
+Concordia is an example solution built on Web3 ID. It allows you to transfer trust between Telegram and Discord. Using verifiable credentials in the |bw|, you can prove ownership of accounts on other platforms. This is done by allowing users to link their accounts and, optionally, real name. You can also ``/check`` and ``/verify`` other users in Telegram and Discord. In other words, if you know John Doe in one platform, you can also trust him in the other platform using his verifiable credentials.
+
+To issue credentials and perform verification that these are your credentials, you can use the Concordia Social media verifier. The Concordia Social media verifier performs all the steps needed for you to allow transfer of trust of your user between platforms.
+
+`Concordia social media verifier on Mainnet <https://concordia.mainnet.concordium.software/>`_
+
+`Concordia social meda verifier on Testnet <https://concordia.testnet.concordium.software/>`_
+
+To get started, get verifiable credentials for your accounts (e.g. one for Discord, one for Telegram) in the **Issue credentials** drop-down.
+
+In the **Verification** drop-down, you prove ownership of the accounts for the verifiable credentials you added to your |bw|.
+
+The Concordia Social media verifier also helps you check that your verification was completed successfully in the **Check verification** drop-down.
+
+If you want to remove your verification, click **Remove verification** in the Concordia Social media verifier.
+
+
+On Mainnet, use the links below:
```

Comment:
Also, we should add name of the bots that will be configured in our channels so people could look here and confirm those are truly our bots.

Telegram - Mainnet - @ConcordiaWeb3IDBot
Telegram - Testnet -  @ConcordiaTestBot

Discord- Mainnet - Concordia#0667
Discord - Testnet -  ConcordiaTestnet#4379